eng/doc/fips/README.md

```diff
@@ -245,9 +250,13 @@ The `requirefips` feature is available since Go 1.21.
 
 FIPS mode is normally determined at runtime, but the `requirefips` build tag can be used to make a program always require FIPS mode and panic if FIPS mode can't be enabled for any reason.
 
+> [!NOTE]
+> Since Go 1.24, one can achieve the same effect by setting `GOFIPS140=latest` when building the program.
+> This is the recommended way to build a program that always requires FIPS mode, as it uses the the same mechanism as upstream Go.
```

Comment:
Should `requirefips` be described by the note rather than `GOFIPS140=latest`? This would emphasize GOFIPS140 more, and be more consistent with the other prior-to-1.24 notes.

Reply:
I've reworded all this secion to make it less `requirefips`-dependent while adding some more info.